meta/trusted-committers.md

```diff
@@ -12,9 +14,9 @@ Adding people off-the-bat encourages more contributions, displays trust, allows
 If you want to become a Collaborator, @ mention one of the [Admins](#admins) in an Issue.
 
 ## Admins
-There are currently a handful of Github Admins for this repository. They hold the ability to add collaborators and modify the repositories nuclear options (delete, rename, etc).
+There are currently a handful of Github Admins for this repository. They hold the ability to add Committers and modify the repositories nuclear options (delete, rename, etc).
```

Comment:
```suggestion
There are currently a handful of Github Admins for this repository. They hold the ability to add collaborators and modify the repositories nuclear options (delete, rename, etc).
```

Reply:
In this case the sentence refers to the _GitHub_ definition of `collaborator`, so I think it makes sense to leave it.